payments/templates/cancel.html

```diff
@@ -1 +1,9 @@
-<p>Sorry, something went wrong</p>
+{% extends 'shop/base.html' %}
+{% block content %}
+    <div class="content-section">
+        <h5>Sorry, something went wrong.</h5>
+        <a href="{% url 'payments:top_up' %}"> Start over</a>
+        <br>
+        <a href="{% url 'shop-home' %}"> Go to the main page</a>
```

Comment:
Moze moglibysmy dodac opcje: Skontaktuj sie z administratorem etc. Mysle, ze to mogloby byc pomocne jak komus platnosci nie przejda kikla razy pod rzad. Ale to moze temat na przepisywanie apki :)

Reply:
skladalam taka propozycje ale Damian potwierdzil ze wystarczy ze wystapil blad :smile: